ecl/eclagent/eclagent.cpp

```diff
@@ -2318,7 +2333,7 @@ void EclAgentWorkflowMachine::noteTiming(unsigned wfid, timestamp_type startTime
     updateWorkunitStat(wu, SSTworkflow, scope, StWhenStarted, nullptr, startTime, 0);
     updateWorkunitStat(wu, SSTworkflow, scope, StTimeElapsed, nullptr, elapsedNs, 0);
 
-    const __int64 cost = calcCost(agent.queryAgentMachineCost(), elapsedNs);
+    const cost_type cost = calcCost(agent.queryAgentMachineCost(), ns2Secs(elapsedNs));
```

Comment:
I think this should also be including the cost for all graphs for this workunit that are executed in thor, so that the cost for a workflow item is the total cost for that workflow item.